docs/reference/commandline/node_update.md

```diff
@@ -19,10 +19,27 @@ Update a node
 Options:
       --availability string   Availability of the node (active/pause/drain)
       --help                  Print usage
+      --label-add value       Add/update node labels (key1=value1,key2=value2)
+      --label-rm value        Remove node labels (key1,key2)
       --membership string     Membership of the node (accepted/rejected)
       --role string           Role of the node (worker/manager)
 ```
 
+### Add a metadata to a node
+
+Add metadata to a node using node labels. When you [create a service](service_create.md),
+you can use node labels as a constraint. A constraint limits the nodes where the
+scheduler deploys tasks for a service.
+
+For example, to add a `type` label to identify nodes where the scheduler should
+deploy message queue service tasks:
+
+```
+docker node update --label-add type=queue worker1
+```
+
+For more information about labels, refer to [apply custom
+metadata](../../userguide/labels-custom-metadata.md).
```

Comment:
@sfsmithcha I think there *may* be confusion (given that both I and @vdemeester were mixing them up); they're both defined for a "node", but one at the actual daemon, and one through the "node" command.